packages/katello/katello/restore.rb

```diff
@@ -264,8 +271,7 @@ def backup_valid?
     end
 
     def confirm
-      puts "WARNING: This script will drop and restore your database."
```

Comment:
I definitely think we should still print this message when its not an incremental backup